src/app/ApplicationTemplate.Shared/Client/Data/DadJokes/DadJokesResponse.cs

```diff
@@ -1,6 +1,6 @@
 ﻿namespace ApplicationTemplate.Client
 {
-	public record DadJokesResponse
```

Comment:
can we leave record? if you change to class, you have to add it to the serialization

Reply:
if deserialization works, let's keep records :)